pallets/interest-accrual/src/lib.rs

```diff
@@ -383,6 +421,23 @@ pub mod pallet {
 			});
 			T::DbWeight::get().reads_writes(count, count)
 		}
+
+		pub fn count_rates() -> Weight {
+			let count = Rate::<T>::iter_keys().count();
+			RateCount::<T>::set(count as u32);
+			T::DbWeight::get().reads(count as u64) + T::DbWeight::get().writes(1)
+		}
```

Comment:
Is this migration-code? 

can we move this into a migration module. Othweise, its not simple to see which code can be removed or so in the future.

Reply:
Moved to hooks and version-checked